test/integration/scan_test.go

```diff
@@ -343,6 +347,17 @@ func TestScanCreateWithThresholdParseError(t *testing.T) {
 func TestScanCreateWithThresholdAndReportGenerate(t *testing.T) {
```

Comment:
why do you change the existing test instead of adding one?

Reply:
Or told me to change just the authentication of one existing test that generates report

Reply:
@tamarleviCm , cause we already have tests for reports, so instead of adding more time to the tests pipeline to reuse them